contrib/mkimage-yum.sh

```diff
@@ -115,14 +120,16 @@ rm -rf "$target"/sbin/sln
 rm -rf "$target"/etc/ld.so.cache "$target"/var/cache/ldconfig
 mkdir -p --mode=0755 "$target"/var/cache/ldconfig
 
-version=
-for file in "$target"/etc/{redhat,system}-release
-do
-    if [ -r "$file" ]; then
-        version="$(sed 's/^[^0-9\]*\([0-9.]\+\).*$/\1/' "$file")"
-        break
-    fi
-done
+if [ -z "$version" ]; then
+    version=
```

Comment:
This line should move up above the `while getopts ...` line to ensure we give the variable the default value we expect it to have (and don't let something bleed in from the environment for example).

Reply:
The default value of `$version` is set from `$target` directory which is constructed by yum command.
So if the line will move up, the `$version` variable will be null by default.
I think this suggest will destroy previous default behavior...

Reply:
Sorry, let me be more clear -- I _just_ mean the `version=` bit should move up, ala:

```bash
install_groups="Core"
version=
while getopts ":y:p:g:t:h" opt; do
￼	# ...
done
# ...
if [ -z "$version" ]; then
￼	for file in "$target"/etc/{redhat,system}-release
# ...
```

Reply:
Ah... OK, I understand.

Reply:
I've moved up to above the `while getopts ...` at 1504dae.

Reply:
oh; did you forget to remove it here? (think it had to be _moved_, not copied)

Reply:
Sorry, I forgot to remove this essential line...
I've removed the line.